README.md

```diff
@@ -42,18 +42,20 @@ This is quite slow and can take up to one or two hours but can provide good resu
 -   Go
 -   PHP
 -   TypeScript
+-   TSX
```

Comment:
Should that really be in that category, or rather unter "Basic support"? Compared to the other languages here, we do not have that many tests for it. On the other hand, TSX is not dramatically different from TS, of course. Mh.

Reply:
I would like to have TSX as a stable language as mentioned in the Issue.

Reply:
Then i will add more tests to make it more stable